doctr/models/recognition/master/pytorch.py

```diff
@@ -0,0 +1,284 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import math
+import torch
+from torch import nn
+from torch.nn import functional as F
+from typing import Dict, Any, Tuple, Optional, List
+
+from ....datasets import VOCABS
+from ...utils import conv_sequence_pt
+from ...backbones import resnet_stage
+from ..transformer import Decoder, positional_encoding, create_look_ahead_mask, create_padding_mask
+from .base import _MASTER
+
+__all__ = ['MASTER']
+
+
+default_cfgs: Dict[str, Dict[str, Any]] = {
+    'master': {
+        'vocab': VOCABS['french'],
```

Comment:
Here I meant that we had to remove `post_processor` but the rest is relevant, the predictor will need the normalization stats and it doesn't hurt keeping the input shape :)